README.md

```diff
@@ -309,8 +309,21 @@ Options to XSecureLock can be passed by environment variables:
     and fall back to XRandR 1.2. Not recommended.
 *   `XSECURELOCK_PAM_SERVICE`: pam service name. You should have a file with
     that name in `/etc/pam.d`.
-*   `XSECURELOCK_PARANOID_PASSWORD`: make `auth_x11` hide the password
-    length.
+*   `XSECURELOCK_PARANOID_PASSWORD`: (deprecated) makes `auth_x11` hide the
```

Comment:
Let's actually remove it from the README.

Also, in the code, can you please add a comment like here https://github.com/google/xsecurelock/blob/6f8876ee0eef0556f15ac06568944cbb20663e1d/helpers/dimmer.c#L291 (yes, _inside_ the string so ./ensure-documented-settings.sh can no longer see it) so I can't forget to remove the old setting in v2.0.0?

Reply:
Fixed
`README.md` - 4ccc9d4
`auth_x11.c` - b1701ec